octoprint_mrbeam/static/css/hopscotch.min.css

```diff
@@ -445,14 +445,15 @@ div.hopscotch-bubble .hopscotch-bubble-close.hide-all {
   display: none;
 }
 div.hopscotch-bubble .hopscotch-bubble-number {
-  background: transparent url(../img/sprite-green.png) 0 0 no-repeat;
```

Comment:
This is a library. We cannot modify it. Please fine another way to do that.

Reply:
This library is very old and will not get any updates as it dropped the development. Thats why I decided to do it here as it will be only till we will use a newer one.

Reply:
I understand this but nevertheless, I would not take that approach and especially not by removing classes and modifying JS